docs/sources/enterprise/access-control/manage-rbac-roles.md

```diff
@@ -4,8 +4,8 @@ menuTitle: 'Manage RBAC roles'
 description: 'Learn how to view permissions associated with roles, create custom roles, and update and delete roles in Grafana.'
 aliases:
   [
-    docs/grafana/latest/enterprise/access-control/manage-role-assignments/,
-    docs/grafana/latest/enterprise/access-control/provisioning/,
+    /docs/grafana/latest/enterprise/access-control/manage-role-assignments/,
```

Comment:
Add the (") where appropriate following this example:
aliases = ["/docs/grafana/latest/manage-users/", "/docs/grafana/latest/manage-users/server-admin/", "/docs/grafana/latest/manage-users/server-admin/server-admin-manage-users/"]

Reply:
good catch, thanks!